### PR TITLE
DOCS-3280 Adds Frustration Signals Documentation

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Renames branch, adds documentation and screenshots for Frustration Signals doc + Dashboard page, and updates the TOC.

### Motivation

DOCS-3280

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 1]
```diff
@@ -246,6 +246,15 @@ Source errors include code-level information about the error. More information a
 | `action.target.name` | string | Element that the user interacted with. Only for automatically collected actions. |
 | `action.name` | string | User-friendly name created (for example, `Click on #checkout`). For [Custom User Actions][15], the action name given in the API call. |
 
+### Frustration signals fields
+
+| Field                | Type   | Description                                                   |
+|-------------------------------|--------|---------------------------------------------------------------|
+| `session.frustration.count`     | number | Count of all frustration signals associated with one session. |
+| `view.frustration.count`        | number | Count of all frustration signals associated with one view.    |
+| `action.frustration.type:dead`  | string | The dead clicks detected by the RUM Browser SDK.              |
```
Comment (jamiemilstein) [inline content/en/real_user_monitoring/browser/data_collected.md:255]: for all of the action.frustration.type they need to have a click after. so for example, it's action.frustration.type:dead_click. this naming convention should follow on the other. two action types as well

File: content/en/real_user_monitoring/frustration_signals/_index.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  trackInteractions: true,
+  trackFrustrations: true
+})
+```
+
+Frustration signals require actions. Enabling `trackFrustrations` automatically enables `trackInteractions`.
+
+## Usage
+
+Frustration signals appear as high-level data points representing sources of user frustration on the [**RUM Applications** page][1]. The [RUM Explorer][2] displays a count of frustration signals as a column in the **List** visualization.
```
Comment (jamiemilstein) [inline content/en/real_user_monitoring/frustration_signals/_index.md:52]: not sure where the second sentence came from, we can remove. it's not true unless a user does this themselves

File: content/en/real_user_monitoring/frustration_signals/_index.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+Frustration signals appear as high-level data points representing sources of user frustration on the [**RUM Applications** page][1]. The [RUM Explorer][2] displays a count of frustration signals as a column in the **List** visualization. 
+
+### Application list
+
+Hover over the list of browser sessions and click on a session to observe a user's frustrated click behavior. Or, click **Frustrated Sessions** to access sessions with a frustration signal. 
+
+### Explore the frustration signals dashboard
+
+The **Frustration Signals** dashboard provides an overview of frustration levels across your application, showing you topics like the most frustrated users and pages with the highest amount of frustration signals.
+
```
Comment (jamiemilstein) [inline content/en/real_user_monitoring/frustration_signals/_index.md:61]: this is where we should add the dashboard, can you use the image i sent over yesterday? that one is what the final will look like

File: content/en/real_user_monitoring/frustration_signals/_index.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+Frustration Count
+: Find sessions and views where any frustration signal occurred. For example, if you want to find any user sessions or views that had at least one frustration signal occur, add `session.frustration.count:>0` or `view.frustration.count:>0` to the search query.
+
+#### Sessions
+
+Click on a session with a value in the **Frustration Count** column to examine the user frustration detected. You can see the type of signal (ranging from `rage click`, `dead click`, or `error click`) and the event timeline that shows what occurred during the session.
+
+#### Views
+
+Click on a view to identify if a user was frustrated with the `frustration detected` tag.
```
Comment (jamiemilstein) [inline content/en/real_user_monitoring/frustration_signals/_index.md:82]: suggesting to reword "Click on a view to identify if a user was frustrated on a specific page with the 'frustration detected' tag

File: content/en/real_user_monitoring/frustration_signals/_index.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+#### Errors
+
+Click on an error in the **Errors** tab to open a side panel with error details. Under the **Parent hierarchy**, you can see if a frustration signal occurred next to the action.
+
+{{< img src="real_user_monitoring/frustration_signals/errors_tab.png" alt="Errors Tab in the Actions side panel" style="width:90%;" >}}
+
+## Watch frustration signals in Session Replay
+
+In [Session Replay][7], you can observe a replication of real user activity. Replays provide video evidence of the actions users take when they exhibit signs of frustration.
```
Comment (jamiemilstein) [inline content/en/real_user_monitoring/frustration_signals/_index.md:106]: add "video-like" before replication

**Review by jamiemilstein — CHANGES_REQUESTED** [review]
left a bunch of comments directly here

File: content/en/real_user_monitoring/frustration_signals/_index.md (hunk 2 above)
Comment (alai97) [reply content/en/real_user_monitoring/frustration_signals/_index.md:52]:
```suggestion
Frustration signals appear as high-level data points representing sources of user frustration on the [**RUM Applications** page][1]. To display a list of frustration signals count in the [RUM Explorer][2], click the **Options** button and add a column with `@session.frustration.count`. 
```

**Comment by buraizu** [comment]
Thanks, created DOCS-3641 to track as an editorial review!

File: content/en/real_user_monitoring/frustration_signals/_index.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+
+The Actions tab displays the `frustration detected` tag if the selected action contains a frustration signal.
+
+If multiple frustration signals occur in an action, they are displayed under **What Happened** in the action panel.
+
+{{< img src="real_user_monitoring/frustration_signals/actions_panel_multiple_frustration_signals.png" alt="Multiple frustration signal types detected in the action under What Happened" style="width:90%;" >}}
+
+#### Errors
+
+Click on an error in the **Errors** tab to open a side panel with error details. Under the **Parent hierarchy**, you can see if a frustration signal occurred next to the action.
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/real_user_monitoring/frustration_signals/_index.md:100]:
```suggestion
Click on an error in the **Errors** tab to open a side panel with error details. You can see if a frustration signal occurred.
```

File: content/en/real_user_monitoring/frustration_signals/_index.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+A session replay's user journey details the events that occur in chronological order. Hover over an event to move to that point in time in the replay, for example, when a dead click occurred.
+
+{{< img src="real_user_monitoring/frustration_signals/session_replay_frustration_signals.png" alt="Frustration signals appear in a browser recording" style="width:90%;" >}}
+
+ For more information, see the [Session Replay documentation][8].
+
+## Create alerts for frustration signals
+
+You can create monitors and set alerts on frustration signals to notify you or your team if any frustration signals occur on an important page in your application.
+
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/real_user_monitoring/frustration_signals/_index.md:117]:
```suggestion

For example, to set an alert that notifies you if any frustration signals occur on a specific page:

```

File: content/en/real_user_monitoring/frustration_signals/_index.md (hunk 3 above)
Comment (alai97) [reply content/en/real_user_monitoring/frustration_signals/_index.md:61]: I included the image in the RUM Frustration Signals Dashboard page already, I don't think we should duplicate the image in two places.

**Review by jamiemilstein — APPROVED** [review]
i think this is good. i may make a few changes in the next week or so but lets ship this!

File: content/en/real_user_monitoring/frustration_signals/_index.md [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+  text: 'Learn about Session Replay'
+---
+
+## Overview
+
+<div class="alert alert-warning">
+For access to frustration signals, contact <a href="/help">Datadog Support</a>.
+</div>
+
+Frustration signals identify the highest points of user friction in your application by surfacing moments where users exhibit frustration.
```
Comment (cswatt) [inline content/en/real_user_monitoring/frustration_signals/_index.md:22]:
```suggestion
Frustration signals identify the highest points of user friction in your application by surfacing moments when users exhibit frustration. 
```

File: content/en/real_user_monitoring/frustration_signals/_index.md [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+<div class="alert alert-warning">
+For access to frustration signals, contact <a href="/help">Datadog Support</a>.
+</div>
+
+Frustration signals identify the highest points of user friction in your application by surfacing moments where users exhibit frustration. 
+
+RUM collects three types of frustration signals:
+
+Rage Clicks
+: A user clicks on an element more than three times in a one-second rolling window.
```
Comment (cswatt) [inline content/en/real_user_monitoring/frustration_signals/_index.md:27]:
```suggestion
: A user clicks on an element more than three times in a one-second sliding window.
```

they mean the same thing, but the other one said "sliding" so let's stick with sliding

File: content/en/real_user_monitoring/frustration_signals/_index.md [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+
+{{< img src="real_user_monitoring/frustration_signals/frustration_signals_in_performance_tab.png" alt="Frustration Signals Actions in the Events dropdown menu in the performance waterfall graph" style="width:90%;" >}}
+
+The performance waterfall displays actions containing frustration signals.
+
+{{< img src="real_user_monitoring/frustration_signals/actions_frustration_signal.png" alt="Frustration signals detected as an action" style="width:90%;" >}}
+
+#### Actions
+
+The Actions tab displays the `frustration detected` tag if the selected action contains a frustration signal.
```
Comment (cswatt) [inline content/en/real_user_monitoring/frustration_signals/_index.md:92]:
```suggestion
The **Actions** tab displays the `frustration detected` tag if the selected action contains a frustration signal.
```

File: content/en/real_user_monitoring/frustration_signals/_index.md [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+
+Click on an error in the **Errors** tab to open a side panel with error details. You can see if a frustration signal occurred.
+
+{{< img src="real_user_monitoring/frustration_signals/errors_tab.png" alt="Errors Tab in the Actions side panel" style="width:90%;" >}}
+
+## Watch frustration signals in Session Replay
+
+In [Session Replay][7], you can observe a video-like replication of real user activity. Replays provide video evidence of the actions users take when they exhibit signs of frustration.
+
+A session replay's user journey details the events that occur in chronological order. Hover over an event to move to that point in time in the replay, for example, when a dead click occurred.
```
Comment (cswatt) [inline content/en/real_user_monitoring/frustration_signals/_index.md:108]:
```suggestion
A session replay's user journey details the events that occur in chronological order. Hover over an event to move to that point in time in the replay: for example, when a dead click occurred.
```

File: content/en/real_user_monitoring/frustration_signals/_index.md [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+
+For more information, see the [Real User Monitoring Monitor documentation][9].
+
+## Troubleshooting
+
+### Why is a rage click not created when a user presses a key (like Delete) on the keyboard?
+
+Frustration signals are generated from mouse clicks, not keyboard strokes. 
+
+### Why does the side panel show a session has a different number of frustration signals that in the event timeline?
```
Comment (cswatt) [inline content/en/real_user_monitoring/frustration_signals/_index.md:130]:
```suggestion
### Why does the side panel show that a session has a different number of frustration signals than in the event timeline?
```

File: content/en/real_user_monitoring/frustration_signals/_index.md [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+
+If a session is live, it is fetching information, and may cause the banners to reflect a different number than those in the timeline.
```
Comment (cswatt) [inline content/en/real_user_monitoring/frustration_signals/_index.md:132]:
```suggestion
If a session is live, it is fetching information and may cause the banners to reflect a different number than those in the timeline.
```

**Review by cswatt — CHANGES_REQUESTED** [review]

**Pull request merged**